<launch>

  <node pkg ="my_turtlebot2"
        type="my_turtlebot2_main.py"
        name="my_turtlebot2_main"
        output="screen">
  </node>
  
  <node pkg="my_turtlebot2"
        type="my_turtlebot2_topics.py"
        name="my_turtlebot2_topics"
        output="screen">
  </node>
  
  <node pkg ="my_turtlebot2"
        type="my_turtlebot2_services_server.py"
        name="my_turtlebot2_services_server"
        output="screen">
  </node>
  
  <node pkg ="my_turtlebot2"
        type="my_turtlebot2_services_client.py"
        name="my_turtlebot2_services_client"
        output="screen">
  </node>

</launch>